src/workspace.ts

```diff
@@ -165,20 +165,20 @@ export class Workspace {
    * wildcard-containing patterns as needed) to return full GitHub repo
    * information for all matched repos.
    */
-  private async _determineGitHubRepos(): Promise<GitHubRepo[]> {
+  async determineGitHubRepos(): Promise<GitHubRepo[]> {
     const matchPatterns = this.options.match;
     const excludePatterns =
-        (this.options.exclude || []).map(String.prototype.toLowerCase);
+        (this.options.exclude ||
+         []).map(((excludePattern) => excludePattern.toLowerCase()));
```

Comment:
I'd almost always prefer this style of callback. It prevents soooo many problems.

Reply:
agreed, I keep re-learning that lesson